app/assets/stylesheets/limber/plate.scss

```diff
@@ -46,12 +46,16 @@
       display: inline-block;
       border: 2px $gray-800 solid;
       border-radius: 7px;
-      font-size: small;
 
```

Comment:
Small is an absolute size, so while shrinking the text on 96 well plates, it was actually making it larger on 384 well plates. This was added for the chromium pipelines, where it helped longer tag indicies fit within the well. 

80% seemed to solve this issue, but made the labels uncomfortably small on 384 well plates. Given any sort of scaling was unnecessary for tags 1-99 I decided to only scale the label when we actually needed to.